README.md

```diff
@@ -7,6 +7,9 @@
 ## Functionality
 metrics-pdns.rb - Query PowerDNS for statistics. Record stats as metrics and output in Graphite plaintext format
 
+arguments:
+--syslog: specify the path to syslog to fetch the extra pdns stats from. default: /var/log/messages
```

Comment:
If we are adding `--syslog` we should add `--extra`

Reply:
yes I need to change this with the new flag I added in the latest commit.